content/general/concepts/programming-paradigms/programming-paradigms.md

```diff
@@ -0,0 +1,20 @@
+---
+Title: 'Programming Paradigms'
+Description: 'An approach to programming. Depending on the language, the preferred paradigm may differ.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Development'
+Tags:
+  - 'Dynamic Programming'
+  - 'Functional Programming'
+  - 'Object Oriented Programming'
```

Comment:
We already have an OOP tag. Adding the tag spelled out is going to affect a lot of entries that already have the OOP tag.